### ISSUE TITLE
autospec requires complete arguments even for '--help'

### ISSUE THREAD

**Issue by jkarni** [hspec/autospec#13]
autospec tries to run even with the `--help` option:

```
$ autospec  --help'                                                                 
listening on .autospec.sock
Ok, modules loaded: none.

<interactive>:28:69:
    Not in scope: ‘spec’
    Perhaps you meant ‘Test.Hspec.hspec’ (imported from Test.Hspec)
```

If I provide e.g. `test/Spec.hs`, it works, but only after loading the files.
